### PR TITLE
Disable gradients in inference mode

### PR DESCRIPTION
**Pull request by aartbik**

### CODE REVIEW

**Review by reidtatge — APPROVED** [review]
You're just disabling the gradient stuff. Oh, for inference mode.  Never mind.  Approved.

**Comment by aartbik** [comment]
> You're just disabling the gradient stuff? Any particular reason why?

No, I am disabling gradients *only* during validation. That's the proper PyTorch setup.

Line 17-28: training, gradients on, model in train mode

Line 31-44: evaluation, gradients off, model in eval mode

**Comment by reidtatge** [comment]
Yeah, I realized that right after I wrote the message.  No problem.

**Comment by aartbik** [comment]
It looks like one of you has to commit this....

**Comment by reidtatge** [comment]
Done.
